components/order/Printer.tsx

```diff
@@ -38,15 +38,13 @@ export class OrderPrinter extends React.PureComponent<{ order: Order }> {
                 <table>
                   <tr>
                     <td>
-                      Fabristic
+                      {brand.name}
                       <br />
-                      The Pirate Cat
+                      Przykładowa, 9/11
```

Comment:
I think we can skip the address and NIP number. I don't want it to look like a real invoice.

Reply:
As you wish it should go to brand config anyways